src/universal/.devcontainer/local-features/machine-learning-packages/install.sh

```diff
@@ -44,11 +50,11 @@ if [[ "$(python --version)" != "" ]] && [[ "$(pip --version)" != "" ]]; then
     install_python_package "matplotlib"
     install_python_package "seaborn"
     install_python_package "scikit-learn"
-    install_python_package "torch"
+    install_python_package "torch" -f https://download.pytorch.org/whl/cpu/torch_stable.html
```

Comment:
```suggestion
    install_python_package "torch" "-f https://download.pytorch.org/whl/cpu/torch_stable.html"
```

Why don't we pass it as ^, then we can avoid using `shift`

Reply:
sure done